src/JustBehave/AsyncBehaviourTestBase.cs

```diff
@@ -85,7 +85,7 @@ protected virtual Layout LogLayout()
             return "${message}";
         }
 
-        protected virtual void PostAssertTeardown() { }
+        protected virtual Task PostAssertTeardownAsync() => Task.FromResult(true);
```

Comment:
`Task.CompletedTask`? Or is that not supported by all the TFMs?

Reply:
Not supported by the net452 tfm, might create an internal static